src/twisted/application/twist/_twist.py

```diff
@@ -84,39 +84,35 @@ def startService(reactor, service):
 
 
     @staticmethod
-    def runnerOptions(twistOptions):
+    def runnerArguments(twistOptions):
```

Comment:
This seems like an odd placement and signature for this method.  We know that these are arguments for `Runner` - shouldn't it just be constructing one, rather than synthesizing keyword arguments which have no other use than to be passed to one?

It strikes me that perhaps this should be `Runner.fromTwistOptions` instead?